cores/esp32/esp32-hal-timer.c

```diff
@@ -220,24 +213,23 @@ void timerEnd(hw_timer_t *timer){
     timer_deinit(timer->group, timer->num);
 }
 
+bool IRAM_ATTR timerFnWrapper(void *arg){
+    void (*fn)(void) = arg;
+    fn();
+
+    // some additional logic or handling may be required here to approriately yield or not
+    return false;
+}
+
 void timerAttachInterrupt(hw_timer_t *timer, void (*fn)(void), bool edge){
     if(edge){
-        log_w("EDGE timer interrupt is not supported! Setting to LEVEL...");
-        edge = false;
+        log_w("EDGE timer interrupt is not supported!");
```

Comment:
I will leave the previous warning log, so user knows that timerAttachInterrupt is set successfully but on LEVEL interrupt. With your change, it can be misunderstood that its not supported so interrupt won't be attached :)

Reply:
Fixed